src/sqlfluff/core/linter/linted_dir.py

```diff
@@ -106,7 +106,7 @@ def stats(self) -> Dict[str, int]:
         )
 
     def persist_changes(
-        self, formatter: Any = None, fixed_file_suffix: str = "", **kwargs
```

Comment:
I change the `**kwargs` option here to more explicit and specific arguments because it was making it hard to understand the code and reason about what it was doing.